docs/src/tutorials/tut_setup.md

```diff
@@ -1,13 +1,320 @@
+[`AdamiPressureExtrapolation`](@ref), which generally produces the best results
+of the implemented methods.
+See [here](@ref boundary_models) for a comprehensive overview over boundary models.
+```jldoctest tut_setup; output = false
+boundary_model = BoundaryModelDummyParticles(tank.boundary.density, tank.boundary.mass,
+                                             state_equation=state_equation,
+                                             AdamiPressureExtrapolation(),
+                                             smoothing_kernel, smoothing_length)
+boundary_system = BoundarySPHSystem(tank.boundary, boundary_model)
+
+# output
+┌──────────────────────────────────────────────────────────────────────────────────────────────────┐
+│ BoundarySPHSystem{2}                                                                             │
+│ ════════════════════                                                                             │
+│ #particles: ………………………………………………… 276                                                              │
+│ boundary model: ……………………………………… BoundaryModelDummyParticles(AdamiPressureExtrapolation, Nothing) │
+│ movement function: ……………………………… nothing                                                          │
+└──────────────────────────────────────────────────────────────────────────────────────────────────┘
+```
+
+## Semidiscretization
+
+The key component of every simulation is the [`Semidiscretization`](@ref),
+which couples all systems of the simulation.
+All methods in TrixiParticles.jl are semidiscretizations, which discretize
+the equations in time to provide an ordinary differential equation that still
+has to be solved in time.
+By providing a simulation time span, we can call [`semidiscretize`](@ref),
+which returns an `ODEProblem` that can be solved with a time integration method.
+```jldoctest tut_setup; output = false, filter = r"ODEProblem with .*"s
+semi = Semidiscretization(fluid_system, boundary_system)
+ode = semidiscretize(semi, tspan)
+
+# output
+ODEProblem with uType RecursiveArrayTools.ArrayPartition...
+```
+
+## Time integration
+
+We use the methods provided by
+[OrdinaryDiffEq.jl](https://github.com/SciML/OrdinaryDiffEq.jl),
+but note that other packages or custom implementations can also be used.
+
+OrdinaryDiffEq.jl supports callbacks, which are executed during the simulation.
+For this simulation, we use the [`InfoCallback`](@ref), which prints
+information about the simulation setup at the beginning of the simulation,
+information about the current simulation time and runtime during the simulation,
+and a performance summary at the end of the simulation.
+We also want to save the current solution in regular intervals in terms of
+simulation time as VTK, so that we can look at the solution in ParaView.
+The [`SolutionSavingCallback`](@ref) provides this functionality.
+To pass the callbacks to OrdinaryDiffEq.jl, we have to bundle them into a
+`CallbackSet`.
```

Comment:
link `CallbackSet` to the corresponding doc page in SciML?

Reply:
Not sure if this is a good idea. These links might break and we have no way to detect that. So we will have broken links in the docs until someone creates an issue to notify us.